AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/FocusElementHighlight.java

```diff
@@ -54,10 +56,22 @@ public void drawElementHighlight(Canvas canvas) {
 
     this.updateWithNewCoordinates();
 
-    this.drawInnerCircle(
-        this.xCoordinate, this.yCoordinate, this.radius, this.paints.get("innerCircle"), canvas);
-    this.drawNumberInCircle(
-        this.xCoordinate, this.yCoordinate, this.tabStopCount, this.paints.get("number"), canvas);
+    if (!isCurrentElement) {
+      this.drawInnerCircle(
+          this.xCoordinate, this.yCoordinate, this.radius, this.paints.get("innerCircle"), canvas);
+      this.drawNumberInCircle(
+          this.xCoordinate, this.yCoordinate, this.tabStopCount, this.paints.get("number"), canvas);
+    }
+
+    if (isCurrentElement) {
+      this.drawInnerCircle(
+          this.xCoordinate,
+          this.yCoordinate,
+          this.radius,
+          this.paints.get("transparentInnerCircle"),
+          canvas);
+    }
```

Comment:
Seems like a natural candidate for an if/else, possibly with the positive state first just because it's a little more intuitive to read:
```suggestion
    if (isCurrentElement) {
      this.drawInnerCircle(
          this.xCoordinate,
          this.yCoordinate,
          this.radius,
          this.paints.get("transparentInnerCircle"),
          canvas);
    } else {
      this.drawInnerCircle(
          this.xCoordinate, this.yCoordinate, this.radius, this.paints.get("innerCircle"), canvas);
      this.drawNumberInCircle(
          this.xCoordinate, this.yCoordinate, this.tabStopCount, this.paints.get("number"), canvas);
    }
```

Reply:
I like it, made the change.